dev/com.ibm.ws.logging.test/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
to follow standard naming convention, this project should be named `com.ibm.ws.logging_test`

Reply:
done, and also done for com.ibm.ws.ras.intrument_test project also.